github/commitstatus_test.go

```diff
@@ -56,6 +56,15 @@ func TestGitHubStatusSuccessMockAPI(t *testing.T) {
 				rateLimitReset: []int{now + 1, now + 3600},
 			},
 		},
+		{
+			name: "Github is down (Gateway timeout) in the first attempt, success in second attempt",
```

Comment:
```suggestion
			name: "Github is flaky (Gateway timeout) in the first attempt, success in second attempt",
```
In a sense, it's not really down if it succeeded on the second attempt.

Reply:
Changed as suggested...